setup.py

```diff
@@ -28,9 +28,7 @@
     package_dir={"": "src"},
     install_requires=[
         "amazon-braket-sdk>=1.35.0",
-        "pennylane>=0.25.1,<=0.27.0",
-        # https://github.com/PennyLaneAI/pennylane/issues/3867
-        "autoray==0.6.0",
+        "pennylane==0.29.1",

```

Comment:
Minimum version that fixes the autoray issue: https://github.com/PennyLaneAI/pennylane/issues/3867#issuecomment-1456756197